distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -123,7 +117,8 @@ type Vector
 
              [1, 2, 3, 4].length
     length : Number
-    length self = Polyglot.get_array_size self.to_array
+    length self =
+        self.to_array.length
```

Comment:
What do existing or new benchmark say about calling `to_array` just to get `length`?

Reply:
No we shouldn't be calling `to_array`.

We should have a `storage` field which should have `at`, `length` and `to_array`.
The `Vector` will then use these allowing us to have make things appears as vectors without copying.

Reply:
Correct, this is now addressed